src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -192,6 +193,7 @@ object RunFirrtlTransformAnnotation extends HasShellOptions {
       case "verilog"  => new VerilogEmitter
       case "mverilog" => new MinimumVerilogEmitter
       case "sverilog" => new SystemVerilogEmitter
+      case "experimental-rtlil" | "rtlil" => RtlilEmitter
```

Comment:
I don't think this is needed here. Afaik this is only used by the old `CompilerAnnotation`.